package.json

```diff
@@ -127,9 +127,9 @@
     "@elastic/ecs": "^8.11.5",
     "@elastic/elasticsearch": "9.0.3",
     "@elastic/ems-client": "8.6.3",
-    "@elastic/eui": "104.1.0",
-    "@elastic/eui-amsterdam": "npm:@elastic/eui@104.1.0-amsterdam.0",
-    "@elastic/eui-theme-borealis": "3.1.0",
+    "@elastic/eui": "105.0.0",
+    "@elastic/eui-amsterdam": "npm:@elastic/eui@105.0.0-amsterdam.0",
+    "@elastic/eui-theme-borealis": "3.2.0",
```

Comment:
A Summary column icon looks broken now. Can you please update it in `src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_summary_column_header.tsx:57`?

<img width="566" height="155" alt="Screenshot 2025-07-14 at 11 22 11" src="https://github.com/user-attachments/assets/05d04182-1613-430d-8b7f-a86aec66aeaf" />